src/css/AboutMission.css

```
@@ -0,0 +1,196 @@
+  gap: 8px;
+  font-size: 1.1rem;
+  overflow: hidden;
+}
+
+.gradient-explore-btn i {
+  transition: all 0.3s ease;
+}
+
+.gradient-explore-btn:hover {
+  background-position: 0 0;
+  background: linear-gradient(to bottom, #fbb147, #f8713e);
```

Biome (1.9.4)

[error] Unexpected shorthand property background after background-position



(lint/suspicious/noShorthandPropertyOverrides)